src/components/TabularReport/ReportTable.js

```diff
@@ -21,18 +21,43 @@ const ReportTable = ({ content: { title, headers, rows } }) => (
         <h1>{title}</h1>
         <table>
             <thead>
-                <tr className="tabular-report__header-row">
+                <tr>
                     {headers.map((header, index) => (
                         <th key={`h-${header}-${index}`}>{header}</th>
                     ))}
                 </tr>
             </thead>
-            <tbody className={'tabular-report__table-body'}>
+            <tbody>
                 {rows.map((cells, index) => (
                     <Row cells={cells} key={`r-${cells[0]}-${index}`} />
                 ))}
             </tbody>
         </table>
+        <style jsx>{`
+            table {
+                border-collapse: collapse;
+            }
+            thead tr {
```

Comment:
There's no need for using `thead` in this selector, the `tr`s rendered by `Row` shouldn't be affected